docs/src/ci-intro.md

```diff
@@ -126,7 +131,7 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: '17'
-    - name: Build & Install
+    - name: Ensure browsers are installed
```

Comment:
Revert this? The step actually triggers build of the project (mvn compile) and then installs the artifacts into local maven cache. It does *not* install browsers, they will be downloaded on first playwright run.